a10_octavia/tests/unit/controller/worker/tasks/test_vthunder_tasks.py

```diff
@@ -0,0 +1,65 @@
+#    Copyright 2020, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+# -*- coding: utf-8 -*-
```

Comment:
Pretty sure we added this cause to solve an encoding issue which was solved by using a different python version.

Let's update setup.py to indicate the only suitable python version is python3 and greater. Then we can remove this cause it didn't even solve the problem we faced.